code/modules/mob/living/simple_animal/abnormality/_abnormality.dm

```diff
@@ -127,6 +136,31 @@
 				if(patrol_path.len)
 					patrol_move(patrol_path[patrol_path.len])
 
+/mob/living/simple_animal/hostile/abnormality/attackby(obj/O, mob/user, params)
+	if(istype(O, /obj/item/reagent_containers))
+		if(status_flags & GODMODE)
+			if(world.time > chemCooldownTimer)
+				var/obj/item/reagent_containers/myContainer = O
+				harvestChem(myContainer, user)
+			else
+				to_chat(user, "<span class='notice'>You may need to wait a bit longer.</span>")
+		else
+			to_chat(user, "<span class='notice'>This doesn't seem like the right time.</span>")
+	return ..()
+
+/mob/living/simple_animal/hostile/abnormality/proc/harvestChem(obj/item/reagent_containers/C, mob/user)
+	if(harvestPhrase == "dummy")
```

Comment:
Instead of making checks like this just make it the default phrase.

Reply:
The default phrase cannot include [user] or [C].

Reply:
You can make use of replace proc. See how arrivals announcement machine works.

Reply:
You can make use of replace proc. See how arrivals announcement machine works.